tools/perf_util/src/main/java/com/msft/send_perf_metrics/JdbcUtil.java

```diff
@@ -1,17 +1,19 @@
 package com.msft.send_perf_metrics;
 
 import java.sql.DriverManager;
+import java.util.Map;
 import java.util.Properties;
 
 public class JdbcUtil {
 	static java.sql.Connection GetConn() throws Exception {
 		try (java.io.InputStream in = App.class.getResourceAsStream("/jdbc.properties")) {
 			if (in == null)
-				throw new RuntimeException("err");
+				throw new RuntimeException("Error reading jdbc properties");
 			Properties props = new Properties();
 			props.load(in);
+			// loading password via env: ORT_PERF_PASSWORD
```

Comment:
the name of the environment variable is configurable, so we don't need to hard code it in the comment :)